apps/condo/domains/ticket/tasks/exportTicketsTask.js

```diff
@@ -0,0 +1,200 @@
+const { getSchemaCtx } = require('@core/keystone/schema')
+const { loadTicketsBatchForExcelExport, loadTicketCommentsForExcelExport } = require('../utils/serverSchema')
+const { ORGANIZATION_COMMENT_TYPE, RESIDENT_COMMENT_TYPE, REVIEW_VALUES } = require('@condo/domains/ticket/constants')
+const { buildExportFile: _buildExportFile, EXCEL_FILE_META } = require('@condo/domains/common/utils/createExportFile')
+const dayjs = require('dayjs')
+const {
+    getHeadersTranslations,
+    EXPORT_TYPE_TICKETS,
+    ticketStatusesTranslations,
+} = require('@condo/domains/common/utils/exportToExcel')
+const { i18n } = require('@condo/domains/common/utils/localesLoader')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { findAllByKey } = require('@condo/domains/common/utils/ecmascript.utils')
+
+const TICKET_COMMENTS_SEPARATOR = '\n' + '—'.repeat(20) + '\n'
+
+const COMMENT_DATE_FORMAT = 'DD.MM.YYYY, HH:mm'
+
+const DATE_FORMAT = 'DD.MM.YYYY HH:mm'
+const HEADER_DATE_FORMAT = 'DD.MM.YYYY'
+
+const EMPTY_VALUE = '—'
+
+const buildReviewValuesTranslationsFrom = (locale) => ({
+    [REVIEW_VALUES.BAD]: i18n('ticket.reviewValue.bad', { locale }),
+    [REVIEW_VALUES.GOOD]: i18n('ticket.reviewValue.good', { locale }),
+})
+
+const renderComment = (comment, locale) => {
+    const createdAt = dayjs(comment.createdAt).format(COMMENT_DATE_FORMAT)
+    const createdBy = comment.userName
+    const userType = comment.userType === RESIDENT ? i18n('Contact', { locale }) : i18n('Employee', { locale })
+    const content = comment.content
+    const filesCount = comment.TicketCommentFiles
+    const filesCountToRender = filesCount > 0 ? `(${i18n('excelExport.ticket.ticketCommentFilesCount', { locale })}: ${filesCount})` : ''
+
+    return `${createdAt}, ${createdBy} (${userType}): «${content}» ${filesCountToRender}`
+}
+
+/**
+ * Converts record obtained from database to JSON representation for file row
+ *
+ * @param ticket
+ * @return {Promise<{clientName, description: string, source: string, operator: (*|string), number, isEmergency: (string), createdAt: *, statusReopenedCounter: string, executor: string, property, classifier: string, details, isWarranty: (string), floorName, place: string, organizationComments: string, deadline: (*|string), entranceName, updatedAt: *, inworkAt: (*|string), completedAt: (*|string), residentComments: string, unitName, reviewComment: (*|string), clientPhone, isPaid: (string), organization, assignee: string, category: string, reviewValue: (*|string), status}>}
+ */
+const convertRecordToFileRow = async ({ task, ticket, indexedStatuses }) => {
+    const { locale, timeZone } = task
+
+    const reviewValuesTranslations = buildReviewValuesTranslationsFrom(locale)
+
+    const comments = await loadTicketCommentsForExcelExport({ ticketIds: [ticket.id] })
+    const renderedOrganizationComments = []
+    const renderedResidentComments = []
+    comments.forEach((comment) => {
+        switch (comment.type) {
```

Comment:
Maybe throw an exception as default case fo switch? If yes, we wont forget to add branch for new case.